docs/internal/Element-Type-Specs.md

```diff
@@ -1,3 +1,5 @@
+_**Note: this document has been superseded by the zola site https://docs.mflow.nz/**_
+
 _Ongoing authoritative reference of Template Question/Element types, including input parameters and response type (shape). Please ensure this document matches the current implementation at all times._
```

Comment:
Delete everything else -- no point have two slightly different versions hanging around.

Reply:
Cool, just deleted and committed